### ISSUE TITLE
Affine transform of RGB image

### ISSUE THREAD

**Issue by jloganolson** [deepmind/dm_pix#50]
Hello! Do you have any idea why this basic translation matrix is affecting each channel of an RGB image differently? 
```python
img = jp.array(PIL.Image.open('test.png').convert('RGB')) / 255.
print(img.shape)
tx = 100.
ty = 0.
translate_matrix = jp.array([
    [1,0,ty],
    [0,1,tx],
    [0,0,1],
])

t_img = pix.affine_transform(img,translate_matrix,mode="constant")
plt.imshow(t_img)
```
Screenshot of output:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/14350126/199113976-a4ed01c1-a1bc-46aa-9474-a53b930dd465.png">

**Comment by claudiofantacci** [comment]
Hi @jloganolson, thanks for reporting this!
Looks a bit strange to me 🤔 I'm looping in @pabloduque0 for this. He might be away atm, but we'll take care to have a look at this asap.

**Comment by jloganolson** [comment]
Thanks! Examples like this in the documentation or comments for translate, rotate, shear, and scale would be super-useful (and would have saved me quite a bit of time).  Bonus: an example of combining them, e.g. should I do translation_matrix @ rotation_matrix.T?

**Comment by pabloduque0** [comment]
Yes, you can indeed combine transformation matrices (with `.dot` or `@` but **no need to transpose**). Bare in mind that this is a plain `affine_transform` so would need to adjust such matrix if you want the original input "centered" after combining a set of transformations ([see an example from TF](https://github.com/keras-team/keras/blob/b80dd12da9c0bc3f569eca3455e77762cf2ee8ef/keras/preprocessing/image.py#L2476)) if that is what you want.

Yes, we can add maybe examples for the different transformations somewhere.

**Comment by pabloduque0** [comment]
Hello @jloganolson ! 

We updated the documentation with a few basic examples for different transformations.

Hope that helps!

**Comment by jloganolson** [comment]
That's awesome - I ended up just brute force learning it but I'm sure this well help the next person who travels this way.

**Comment by claudiofantacci** [comment]
Thanks @pabloduque0 for the PR and thanks @jloganolson for flagging this!

As this is now clarified and https://github.com/deepmind/dm_pix/pull/51 is merged, I'm closing this issue :smile:
